src/main/scala/com.snowplowanalytics.sauna/SaunaSettings.scala

```diff
@@ -34,17 +35,22 @@ import responders._
  */
 case class SaunaSettings(
   // Loggers
-  amazonDynamodbConfig: Option[AmazonDynamodbConfig],
-  hipchatLoggerConfig: Option[loggers.HipchatConfig],
+  amazonDynamodbConfig: Option[AmazonDynamodbConfig_1_0_0],
+  hipchatLoggerConfig: Option[loggers.HipchatConfig_1_0_0],
 
   // Responders
-  optimizelyConfig: Option[OptimizelyConfig],
-  sendgridConfig: Option[SendgridConfig],
-  hipchatResponderConfig: Option[responders.HipchatConfig],
-  slackConfig: Option[SlackConfig],
-  pagerDutyConfig: Option[PagerDutyConfig],
+  optimizelyConfig: Option[OptimizelyConfig_1_0_0],
+  sendgridConfig_1_0_0: Option[SendgridConfig_1_0_0],
+  sendgridConfig_1_0_1: Option[SendgridConfig_1_0_1],
+  hipchatResponderConfig: Option[responders.HipchatConfig_1_0_0],
+  slackConfig: Option[SlackConfig_1_0_0],
+  pagerDutyConfig: Option[PagerDutyConfig_1_0_0],
 
   // Observers
-  localFilesystemConfigs: List[LocalFilesystemConfig],
-  amazonS3Configs: List[AmazonS3Config],
-  amazonKinesisConfigs: List[AmazonKinesisConfig])
+  localFilesystemConfigs: List[LocalFilesystemConfig_1_0_0],
+  amazonS3Configs: List[AmazonS3Config_1_0_0],
+  amazonKinesisConfigs: List[AmazonKinesisConfig_1_0_0])
+
+object SaunaSettings {
+  def apply(): SaunaSettings = SaunaSettings(None, None, None, None, None, None, None, None, Nil, Nil, Nil)
```

Comment:
Replaces the two huge constructor calls in `IntegrationTests` that had to be modified every time `SaunaSettings` was changed.